src/kem/kyber/arithmetic.rs

```diff
@@ -1,6 +1,16 @@
 use super::constants::{COEFFICIENTS_IN_RING_ELEMENT, FIELD_MODULUS};
 
-pub(crate) type KyberFieldElement = i32;
+/// Values having this type hold a representative 'x' of the Kyber Field.
+/// Let 'fe' be the shorthand for this type
+pub(crate) type FieldElement = i32;
```

Comment:
I wonder if we could go one step further and make this an anonymous struct, so that the Rust type system enforces these types. But for this PR, this is already good documentation.

Reply:
I'll slowly try and make them structs as well, the main thing is we'd need traits/multiple type definitions/some other way to hold all these types in RingElements